blivet/formats/luks.py

```diff
@@ -245,7 +238,7 @@ def _create(self, **kwargs):
                                     key_file=self._key_file,
                                     cipher=self.cipher,
                                     key_size=self.key_size,
-                                    min_entropy=self.min_luks_entropy)
+                                    min_entropy=luks_data.min_entropy)
```

Comment:
This all needs to stay as is if we want to support per-device (more precisely per-format) entropy threshold.